client/blocks/login/continue-as-user.jsx

```diff
@@ -104,38 +91,14 @@ export default function ContinueAsUser( {
 				>
 					{ translate( 'Continue' ) }
 				</Button>
-			</div>
-		);
-	}
-
-	if ( isBlazePro ) {
-		return (
-			<div className="continue-as-user">
-				<div className="continue-as-user__user-info">
-					{ gravatarLink }
-					<div className="continue-as-user__not-you">
-						<Button
-							variant="tertiary"
-							id="loginAsAnotherUser"
-							className="continue-as-user__change-user-link"
-							onClick={ onChangeAccount }
-						>
-							{ translate( 'Sign in as a different user' ) }
-						</Button>
-					</div>
-				</div>
 				<Button
-					variant="primary"
-					className="continue-as-user__continue-button"
-					isBusy={ validatingPath }
-					href={ validatedPath || '/' }
-					__next40pxDefaultSize
+					variant="link"
+					id="loginAsAnotherUser"
+					className="continue-as-user__change-user-link"
+					onClick={ onChangeAccount }
 				>
-					{ `${ translate( 'Continue as', {
-						context: 'Continue as an existing WordPress.com user',
-					} ) } ${ userName }` }
+					{ translate( 'Login with a different account' ) }
```

Comment:
I think we should split Login:

```suggestion
					{ translate( 'Log in with a different account' ) }
```

Reply:
Handled in [`e72b2e6` (#105240)](https://github.com/Automattic/wp-calypso/pull/105240/commits/e72b2e68b2999ed697e8a032ed9bbc1313e21db3)